app/main/db/queries.py

```diff
@@ -247,7 +247,7 @@ def _build_series_view_query(series_id, filters, sorting_orders):
         query = query.order_by(
             sub_query.c.transferring_body,
             sub_query.c.series,
-            sub_query.c.consignment_reference,
+            desc(sub_query.c.last_record_transferred),
```

Comment:
what prompted the change from default sorting of consignment reference asc to last_rec_tasnf desc?

Reply:
consignment_reference sort was set earlier in a query, as the consignment view now requires default sort by record transferred, so amended the query accordingly